examples/examples/custom_type_derives.rs

```diff
@@ -2,8 +2,7 @@
 // This file is dual-licensed as Apache-2.0 or GPL-3.0.
 // see LICENSE for license details.
 
-//! Example verified against polkadot 0.9.18-4542a603cc-aarch64-macos.
-
+//! Example verified against polkadot polkadot 0.9.25-5174e9ae75b.
```

Comment:
I reckon that it would be useful to add an example for `overriding types` as well not really related to this PR but could be part of this example perhaps....

Reply:
Let's add an issue to do this :)

Reply:
Actually I'll do it if you haven't already; I have a list of issues to make anyway :)